src/routes/(logged_in)/+layout.server.ts

```diff
@@ -10,7 +10,8 @@ const facts = [
   'Eludris is factually better than Disco- well, maybe not yet',
   'I am the storm that is approaching',
   'Coffee is kinda pog',
-  'The name pengin originally started as a joke about linux users while watching <a href="https://cdn.eludris.gay/static/pengin.mp4" target="_blank">this video</a> on loop'
+  'The name pengin originally started as a joke about linux users while watching <a href="https://cdn.eludris.gay/static/pengin.mp4" target="_blank">this video</a> on loop',
+  'ggVG is a sensible keybind.'
```

Comment:
Would a trailing comma make sense as to not require two line changes for additional validations of objectively incorrect users?